adoptingerlang.org

```diff
@@ -1980,19 +2027,19 @@ _build/prod/rel/service_discovery
     └── V2__Add_updated_at_trigger.sql
 #+END_SRC
 
-There are no symlinks under =lib= and OTP applications like =stdlib-3.8.1= are included. And at the top of the tree is =erts-10.3.3= which contains the Erlang runtime.
+There are no symlinks under =lib= and OTP applications like =stdlib-3.8.1= are included. At the top of the tree is =erts-10.4.4= which contains the Erlang runtime.
 
 To build the target system of the release we run the =tar= command in the =prod= profile:
 
 #+BEGIN_SRC shell
 $ rebar3 as prod tar
 #+END_SRC
 
-Now we have a tarball =_build/prod/rel/service_discovery/service_discovery-c9e1c805d57a78d9eb18af1124962960abe38e70.tar.gz= that can be copied to a target and run.
+Now we have a tarball =_build/prod/rel/service_discovery/service_discovery-c9e1c805d57a78d9eb18af1124962960abe38e70.tar.gz= that can be copied to any compatible host, unpacked and run.
 
 *** Runtime Configuration
 
-Running a release involves two levels of configuration. First there is the underlying Erlang virtual machine's configuration. Its values have to be set before the VM has started, so they cannot be part of a configuration file like =sys.config=, which requires a running VM to read and parse the Erlang term file. Instead, the VM arguments are passed directly to the =erl= command. To simplify this the =erl= command has an =-args_file= argument to allow command-line arguments to be read from a plain text file. This file is commonly named =vm.args=.
+In the =service_discovery= project we have two files for configuration under the =config/= directory, =vm.args.src= and =sys.config.src=, that are included in the release. These files act as templates to be filled in at runtime based on environment variables. There are two separate files because running a release involves two levels of configuration. First, there is the underlying Erlang virtual machine's settings. These values have to be set before the VM has started, so they cannot be part of a term file like =sys.config= which requires a running VM to read and parse the Erlang term file. Instead, the VM arguments are passed directly to =erl= -- =erl= being the command used to boot our release. To simplify this the =erl= command has an =-args_file= argument to allow command-line arguments to be read from a plain text file. This file is commonly named =vm.args=.
```

Comment:
```suggestion
In the =service_discovery= project we have two files for configuration under the =config/= directory that are included in the release: =vm.args.src= and =sys.config.src=. These files act as templates to be filled in at runtime based on environment variables. There are two separate files because running a release involves two levels of configuration. First, there is the underlying Erlang virtual machine's settings. These values have to be set before the VM has started, so they cannot be part of a term file like =sys.config= which requires a running VM to read and parse the Erlang term file. Instead, the VM arguments are passed directly to =erl= -- =erl= being the command used to boot a release. To simplify this, the =erl= command has an =-args_file= argument to allow command-line arguments to be read from a plain text file. This file is commonly named =vm.args=.
```